pkg/templates/multiclusterhub/base/clusterrole-admin-aggregate.yaml

```
@@ -27,8 +27,17 @@ rules:
   resources: ["klusterletaddonconfigs"]
   verbs: ["create", "get", "list", "watch", "update", "delete", "deletecollection", "patch"]
 - apiGroups: ["wgpolicyk8s.io"]
-  resources: ["policyreports"]
+  resources: ["policyreports", "clusterpolicyreports"]
   verbs: ["get", "list", "watch"]
 - apiGroups: ["submarineraddon.open-cluster-management.io"]
   resources: ["submarinerconfigs", "submarinerconfigs/status"]
   verbs: ["create", "get", "list", "watch", "update", "delete", "deletecollection", "patch"]
+- apiGroups: ["mutations.gatekeeper.sh"]
+  resources: ["*"]
+  verbs: ["create", "get", "list", "watch", "update", "delete", "deletecollection", "patch"]
+- apiGroups: ["admissionregistration.k8s.io"]
+  resources: ["validatingadmissionpolicybindings"]
+  verbs: ["create", "get", "list", "watch", "update", "delete", "deletecollection", "patch"]
+- apiGroups: ["kyverno.io"]
+  resources: ["policies", "clusterpolicies"]
+  verbs: ["create", "get", "list", "watch", "update", "delete", "deletecollection", "patch"]
```

I could be swayed on Gatekeeper since Gatekeeper is included in an ACM subscription, but I'm not sure these permissions should allow anything other than a fetch for Kyverno objects since ACM doesn't include/support Kyverno.

```suggestion
- apiGroups: ["kyverno.io"]
  resources: ["policies", "clusterpolicies"]
  verbs: ["get", "list", "watch"]
```

---

good point! Who should I ask this?

---

You could ask @gparvin?

---

@gparvin  you are right. We need list, get (+watch maybe) to display on discovered policy table

---

@yiraeChristineKim Did we need more than get/list/watch on the other resources?  Make sure you are requesting the least amount of permissions possible to reduce the possibility of security issues in this area.

---

We need get, list, and watch, but I primarily want to ensure compatibility with [this PR](https://github.com/stolostron/multiclusterhub-operator/pull/1723). I am not entirely sure how the ClusterRole is being used